tests/tasks/remove_test_interfaces_with_dhcp.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Remove test interfaces
+  shell: |
+    ip link delete {{dhcp_interface1}}
+    ip link delete {{dhcp_interface2}}
+    ip link delete testbr
+
+    # Remove udev rule for NM to see veth devices starting with test*.....
+    rm -rf /etc/udev/rules.d/88-veth.rules
+    udevadm control --reload-rules
+    udevadm settle --timeout=5
+
+
+- name: Stop dnsmasq/radvd services
+  shell: |
+    pkill -F /run/dhcp_testbr.pid
+    rm -rf /run/dhcp_testbr.pid
+    rm -rf /run/dhcp_testbr.lease
+    if grep 'release 6' /etc/redhat-release; then
```

Comment:
Is this test run only on rhel and centos?

Reply:
No, but on Fedora the file does not exist, so the `else` block is used, there.

Reply:
The file does exist on Fedora:
```
>cat /etc/fedora-release 
Fedora release 32 (Thirty Two)
>cat /etc/redhat-release 
Fedora release 32 (Thirty Two)
>ls -alrtF /etc/redhat-release
lrwxrwxrwx. 1 root root 14 Jun  5 14:00 /etc/redhat-release -> fedora-release
```
So t implicitly skips this test on Fedora, rather than explicitly specifying that the test should be skipped on Fedora.

Reply:
ah, ok, so the file exists but does not contain `release 6` on Fedora as long as Fedora 60 is not released. The test is not skipped on Fedora, it will just use a different setup.